frontend/src/components/modals/AddTaskModal.module.scss

```
@@ -0,0 +1,4 @@
+textarea {
+  min-height: 5em !important;
+  height: 20em;
+}
```

Maybe still add the max-height back in, so a scrollbar won't appear in the modal on resize 👍 
```suggestion
  height: 20em;
  max-height: 30em;
}
```